tests/e2e/specs/setup-mc/step-3-confirm-store-requirements.test.js

```diff
@@ -0,0 +1,456 @@
+						await expect( confirmButton ).toBeVisible();
+						await expect( confirmButton ).toBeEnabled();
+					}
+				} );
+
+				test( 'should see the correct link in each toggled panel', async () => {
+					const panels =
+						storeRequirements.getPrelaunchChecklistPanels();
+
+					const panel1 = await panels.nth( 0 );
+					const link1 = panel1.getByRole( 'link' );
+					await expect( link1 ).toBeVisible();
+					await expect( link1 ).toHaveAttribute(
+						'href',
+						'https://woocommerce.com/document/google-listings-and-ads/compliance-policy/#store-is-live'
+					);
+
+					const panel2 = await panels.nth( 1 );
+					const link2 = panel2.getByRole( 'link' );
+					await expect( link2 ).toBeVisible();
+					await expect( link2 ).toHaveAttribute(
+						'href',
+						'https://woocommerce.com/document/google-listings-and-ads/compliance-policy/#complete-checkout'
+					);
+
+					const panel3 = await panels.nth( 2 );
+					const link3 = panel3.getByRole( 'link' );
+					await expect( link3 ).toBeVisible();
+					await expect( link3 ).toHaveAttribute(
+						'href',
+						'https://woocommerce.com/document/google-listings-and-ads/compliance-policy/#complete-checkout'
+					);
+
+					const panel4 = await panels.nth( 3 );
+					const link4 = panel4.getByRole( 'link' );
+					await expect( link4 ).toBeVisible();
+					await expect( link4 ).toHaveAttribute(
+						'href',
+						'https://woocommerce.com/document/google-listings-and-ads/compliance-policy/#refund-and-terms'
+					);
+				} );
+
+				test( 'should have the checkbox checked by clicking "Confirm" button', async () => {
+					// Click "Confirm" button from panel 1 and panel 2
+
+					const panels =
+						storeRequirements.getPrelaunchChecklistPanels();
+					const checkboxes =
+						storeRequirements.getPrelaunchChecklistCheckboxes();
+
+					for ( let i = 0; i <= 1; i++ ) {
+						// Using .first() because after clicking "Confirm" the row that was clicked would disappear,
+						// the second panel would become the first one at the second iteration of for loop.
+						const panel = await panels.first();
+
+						// Did not use .first() for the checkboxes because checkboxes would not disappear after clicking "Confirm".
+						const checkbox = await checkboxes.nth( i );
+
+						const confirmButton = panel.getByRole( 'button', {
+							name: 'Confirm',
+							exact: true,
+						} );
+
+						await confirmButton.click();
+
+						await expect( checkbox ).toBeDisabled();
+						await expect( checkbox ).toBeChecked();
+					}
+				} );
+
+				test( 'should have the checkbox checked by clicking the checkbox', async () => {
+					// Click checkboxes from panel 3 and panel 4
+
+					const checkboxes =
+						storeRequirements.getPrelaunchChecklistCheckboxes();
+
+					for ( let i = 2; i <= 3; i++ ) {
+						const checkbox = await checkboxes.nth( i );
+						await checkbox.click();
+						await expect( checkbox ).toBeDisabled();
+						await expect( checkbox ).toBeChecked();
+					}
```

Comment:
What do you think if we test whether the following requests are being made: `POST wc/gla/mc/settings` and, when we click "Continue," we also send the request `POST wc/gla/mc/contact-information`? Should we verify this one as well?